versioned_docs/version-latest/04-winglibs/04-toc.md

```diff
@@ -0,0 +1,45 @@
+---
+title: List of winglibs
+id: all-winglibs
+sidebar_label: List of winglibs
+description:  Table of all Wing libraries
+keywords: [winglib, Wing library]
+---
+
+| Library  | Package name  | Version | Description | Supported cloud providers | Supported Wing platforms |
+| -------- | ------- | ------- | ------- | ------- | ------- |
```

Comment:
I'm wondering if the two separate columns (supported cloud providers and supported platforms) might confuse people - if there's mulitple AWS platforms but each one links to what it does / means, maybe that's good enough?

Reply:
Yeah it's a good point TBH. I just worry a bit about new folks coming to the platform not understanding platforms, and how they map to providers, but I think I'm over thinking this tbh.

Coming back at it now after a break over the weekend, I think you are right, I will remove the the cloud provider stuff and just keep platforms 🙏